.github/workflows/go.yml

```diff
@@ -14,6 +14,8 @@ defaults:
 jobs:
   main:
     runs-on: ubuntu-18.04
+    env:
+      built_image: "quay.io/metallb/metallb-operator:ci" # Arbitrary name
```

Comment:
I _don't think_ you need to specify a full registry name here.
metallb-operator:ci should work and cause less confusion

Reply:
done